includes/admin/class-wc-rest-payments-settings-controller.php

```diff
@@ -230,6 +233,85 @@ public function validate_statement_descriptor( $value, $request, $param ) {
 		return true;
 	}
 
+	/**
+	 * Validate the business support email.
+	 *
+	 * @param array           $value The value being validated.
+	 * @param WP_REST_Request $request The request made.
+	 * @param string          $param The parameter name, used in error messages.
+	 * @return true|WP_Error
+	 */
+	public function validate_business_support_email_address( $value, $request, $param ) {
```

Comment:
IMHO, the validate_* methods should ideally be private and not require doc comments.

Reply:
Validation methods are public to be tested with the Unit Tests.

Reply:
I wouldn't do a method public just because we need to test it. You can use reflection ( not a nice way to test things, I know, but in some cases we can use it ), or the second choice, to use `WP_REST_Request` and call the route with invalid parameters, in our case the email address. I would go with the second choice.

Reply:
@dpaun1985 I wonder if it is worth it to make the validation methods private in the above case? The `update_settings` request has a huge amount of props to be checked which will make the tests huge and hard to debug.

Reply:
Since there are a lot of validation methods, we can leave it also public . Both approaches work.